packages/sql/test/__snapshots__/fixtures.spec.ts.snap

```diff
@@ -1,16 +1,14 @@
 // Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
 
 exports[`parser and printer > should format all fixtures > 144.sql 1`] = `
-"CREATE TABLE
-  \\"test\\" (\\"id\\" uuid NOT NULL)
+"CREATE TABLE \\"test\\" (\\"id\\" uuid NOT NULL)
 WITH
   (oids = false);
 "
 `;
 
 exports[`parser and printer > should format all fixtures > 233.sql 1`] = `
-"CREATE TABLE
-  \\"test\\" (\\"id\\" uuid NOT NULL)
+"CREATE TABLE \\"test\\" (\\"id\\" uuid NOT NULL)
```

Comment:
these are the new format of `CREATE TABLE` mentioned in [the `sql-formatter@15.0.0` release notes](https://github.com/sql-formatter-org/sql-formatter/releases/tag/v15.0.0):

> New formatting style of CREATE TABLE (sql-formatter-org/sql-formatter#495) Thanks to Timon Jurschitsch